.github/workflows/test-java.yaml

```diff
@@ -112,13 +112,13 @@ jobs:
 
   java-example:
     name: Example project – JavaThemis
-    runs-on: ubuntu-20.04
+    runs-on: ubuntu-22.04
     steps:
       - name: Install system dependencies
         run: |
           sudo sh -c 'echo "DEBIAN_FRONTEND=noninteractive" >> /etc/environment'
           sudo apt update
-          sudo apt install --yes gcc make libssl-dev default-jdk
+          sudo apt install --yes gcc make libssl-dev default-jdk dpkg-dev
```

Comment:
for what we need dpkg-dev?

Reply:
For dpkg-architecture. I tested the solution on a real OS installed on Digital Ocean.